.gitignore

```diff
@@ -13,3 +13,8 @@
 
 /node_modules
 /yarn-error.log
+
+# Ignore all of coverage except for last_run.json
```

Comment:
Why?

Reply:
It's what we do on all of our other projects.

Reply:
oh